sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -330,26 +273,114 @@ static struct snd_soc_card sof_audio_card_rt5682 = {
 	.late_probe = sof_card_late_probe,
 };
 
+static const struct x86_cpu_id baytrail_cpu_ids[] = {
+	{ X86_VENDOR_INTEL, 6, INTEL_FAM6_ATOM_SILVERMONT },
+	{}
```

Comment:
you probably want to add CherryTrail so that we can test with the Up board (not Up2)